### PR TITLE
Discovery Bid Adapter: add adUnitCode, gpid, and adslot

### PR DESCRIPTION
**Pull request by ecoeco163**
## Type of change

- [ ] Bugfix
- [x] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes

- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change
When initiating a bid request, new parameters including adUnitCode, adslot, gpid, and token have been added.

### CODE REVIEW

**Review by BaronJHYu — APPROVED** [review]

**Comment by ecoeco163** [comment]
Hello, I am the new developer of Discovery Bid Adapter, there are two checks in my submission that have not automatically run and need to be approved, thank you.

**Comment by ecoeco163** [comment]
Hello, I am the new developer of Discovery Bid Adapter, there are two checks in my submission that have not automatically run and need to be approved, thank you.

**Comment by ecoeco163** [comment]
Hi,  there are two checks in my submission that have not automatically run and need to be approved. Can you please help me with approval and review?thank you @patmmccann  🙌

**Comment by ecoeco163** [comment]
Hi, this PR is important to us. Could you please review it? Thank you very much. @peixunzhang @ChrisHuie 🙌

**Review by raiozhang — APPROVED** [review]
These change are mainly to get more parameters

**Comment by ChrisHuie** [comment]
@ecoeco163 catching up on prs. We had our Prebid Summit this week so most folks have been out this week and are just getting back. Should get reviewed over the next couple of days.

Can you please add tests 🙏

**Comment by ecoeco163** [comment]
Hi, I have added tests. Could you please review it again? Thank you very much.  @ChrisHuie

File: modules/discoveryBidAdapter.js [hunk 1]
```diff
@@ -211,6 +211,60 @@ const popInAdSize = [
   { w: 336, h: 280 },
 ];
 
+/**
+ * get screen size
+ *
+ * @returns {string} eg: "widthxheight"
+ */
+function getScreenSize() {
```
[the remark this answers is not in the document]
Comment (ecoeco163) [reply modules/discoveryBidAdapter.js:219]: Hi, thank you for your suggestion, I used parseSizesInput utils function to handle ScreenSize. Could you please review it again? Thank you very much. 🙏 @ChrisHuie

[the remark this answers is not in the document]
Comment (ecoeco163) [reply modules/discoveryBidAdapter.js:219]: Hi, The PR has passed all the tests, Could you please approve it?  I'm very sorry for being so urgent, but it's really important to us. Thank you @ChrisHuie

**Review by ChrisHuie — APPROVED** [review]

**Pull request merged**